components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3367,27 +3359,24 @@ subroutine ice_deposition_sublimation(qitot_incld, nitot_incld, t, &
 
    return
 
-   ! AaronDonahue: General comment.  In testing these two approaches using the
-   ! p3 standalone I found that qidep is around two orders of magnitude greater for the
-   ! new approach than for the old approach. If we look at the expressions, in
-   ! the old approach we use qvs-dumqvi instead of qv-dumqvi.  qv is 3 or 4
-   ! times larger than qvs which contributes to this huge difference.  Should we
-   ! be using qv?
-
 end subroutine ice_deposition_sublimation
 
 
-subroutine evaporate_sublimate_precip(qc_incld,qr_incld,nr_incld,t,pres, &
-aaa,xx,dt,ab,epsr,epsc,qv,sup,ssat,xxlv, &
+subroutine evaporate_sublimate_precip(qc_incld,qr_incld,nr_incld,qitot_incld,t,pres, &
+lcldm,rcldm,qvs,aaa,xx,dt,ab,epsr,epsc,qv,sup,ssat,xxlv, &
```

Comment:
Kind of an obvious point, but once you delete orig_version, you can delete a lot of variables from this interface.

Reply:
Absolutely.  But always good to note.